configure.ac

```diff
@@ -32,6 +32,12 @@ AC_DEFINE_SUBST(CONF_FALLBACKSTART, "false", [make optirun start applications no
 AC_DEFINE_SUBST(CONF_VGLCOMPRESS, "proxy", [vglclient transport method])
 AC_DEFINE_SUBST(CONF_TURNOFFATEXIT, "false", [state of card when shutting off daemon])
 
+# OpenSUSE: /usr/bin/X -> /var/lib/X11/X -> /usr/bin/Xorg
+# Fedora, Arch Linux: /usr/bin/X -> /usr/bin/Xorg
+# Ubuntu: /usr/bin/X is a custom binary doing authorization and then executes
+#         /etc/X11/X -> /usr/bin/Xorg
+AC_DEFINE_SUBST(CONF_XORG_BINARY, "Xorg", [Xorg binary to run])
```

Comment:
What about using AC_DEFINE_CONF instead, so that it can be changed via ./configure ?

```
# Debian/Ubuntu and Xorg >= 1.18: /usr/bin/Xorg is a script that will run either the normal
#                                                       Xorg binary from /usr/lib/xorg/Xorg, or a SETUID
#                                                       wrapper at /usr/lib/xorg/Xorg.wrap
AC_DEFINE_CONF(CONF_XORG_BINARY, [Xorg binary to run], [
if test "x" = "x$CONF_XORG_BINARY"; then
	CONF_XORG_BINARY=Xorg
fi
])
```

This is very important especially for packagers. WIth Xorg >= 1.18, the binary moves and there's a SETUID wrapper binary that forks into Xorg, and this is not supported by bumblebee. So being able to configure the binary at build time is very handy for Debian/Ubuntu packages.